utils/ratlib.py

```diff
@@ -128,3 +159,73 @@ def try_parse_uuid(suspect: str) -> UUID:
 
     else:
         return result
+
+
+# color functions
+def color(text: str, text_color: Colors, bg_color=None) -> str:
+    """
+    Colorizes the given string with the specified color.
+    Args:
+        text: The text to colorize
+        text_color: a Colors.COLOR
+        bg_color: if specified, the background color
+
+    Returns:
+        str: colorized string
+    """
+    if bg_color is not None:
+        bg_color_code = bg_color.replace('\x03', '')
+        return f'{text_color},{bg_color_code}{text}{Colors.FORMAT_RESET}'
```

Comment:
you would need to preceed this with `FORMAT_COLOR`, but would still be cleaner IMHO